dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsApiServiceImpl.java

```diff
@@ -197,20 +202,43 @@ public Response toolsIdVersionsVersionIdTypeTestsGet(String type, String id, Str
         }
 
         // The getFileType version never returns *TEST_JSON filetypes.  Linking CWL_TEST_JSON with DOCKSTORE_CWL and etc until solved.
-        boolean plainTextResponse = value.getAcceptableMediaTypes().contains(MediaType.TEXT_PLAIN_TYPE) || type.contains("plain");
-
+        boolean plainTextResponse =
+                value.getAcceptableMediaTypes().contains(MediaType.TEXT_PLAIN_TYPE) || type.toLowerCase().contains("plain");
+        ObjectMapper mapper = new ObjectMapper();
         switch (fileType) {
         case CWL_TEST_JSON:
         case DOCKSTORE_CWL:
-            return getFileByToolVersionID(id, versionId, CWL_TEST_JSON, null,
-                    plainTextResponse);
+            List<ToolTests> cwlToolTests = new ArrayList<>();
```

Comment:
This seems repetitive, see if getFileByToolVersionID can be modified/upgraded

Reply:
Fixed